modules/installation-azure-regions.adoc

```diff
@@ -5,6 +5,32 @@
 [id="installation-azure-regions_{context}"]
 = Supported Azure regions
 
-The list of available Microsoft Azure regions is dynamically generated.
+The installation program dynamically generates the list of available Microsoft Azure regions based on your subscription. The following Azure regions were tested in {product-title} version 4.2:
 
-//IDK if there are limits. I have a question out about this.
+* centralus (Central US)
+* eastus (East US)
+* eastus2 (East US 2)
+* northcentralus (North Central US)
+* southcentralus (South Central US)
+* westcentralus (West Central US)
+* westus (West US)
+* westus2 (West US 2)
+* uksouth (UK South)
+* ukwest (UK West)
+* francecentral (France Central)
+* francesouth (France South)
```

Comment:
I did not have the proper quota for francesouth, so we should remove it for now.